### ISSUE TITLE
Auto height issue

### ISSUE THREAD

**Issue by matiosfree** [naver/egjs-flicking#439]
## Description
How to make the slider responsive with auto height?

I have a dynamic content inside the slider. There might be slides with different height. if the content is very high I don't see part of the content. My slider is in the Dialog. Code looks like:

```
const flicking = React.useRef(null);

React.useEffect(() => {
    if (flicking.current) {
      flicking.current.resize();  //Seems it doesn't help also
    }
}, [props.slots]);

return (
  <Flicking
        style={{height: "100%"}} //without that style I don't see content at all
        gap={20}
        bound
        ref={flicking}
        lastIndex={60}
        infinite
        infiniteThreshold={400}
        onNeedPanel={handleNeedPanel}
        resizeOnContentsReady
      >
          {props.slots.map(slot => <div style={{width: 'calc(33% - 13px)'}}>some high content</div>)}
  </Flicking>
)
```

As you can see on the screenshot, there is not any scrollbar and part of the content is just hidden

![27be7da144](https://user-images.githubusercontent.com/3754984/113504439-c3c75c00-9540-11eb-827f-92974a0dd019.jpg)

is there any ideas how it could be improved?

**Comment by WoodNeck** [comment]
Hello, @matiosfree!
Hmm, it's weird that there's no CSS `height` property on the ".eg-flick-viewport" element.
It happens when there're no panels inside, but as there're panels correctly placed so that should not be the case.
I guess the `resize()` is not calling properly.
Can you show me the console result of the below code?
```diff
React.useEffect(() => {
    if (flicking.current) {
+     console.log(flicking.current.getCurrentPanel(), flicking.current.getPanel(0) && flicking.current.getPanel(0).getBbox());
      flicking.current.resize();  //Seems it doesn't help also
    }
}, [props.slots]);
```

**Comment by matiosfree** [comment]
The output is:

```
null null
null {x: 504, y: 106.734375, width: 134.828125, height: 511}
null {x: 504, y: 106.5, width: 134.828125, height: 511}
null {x: 504, y: 106.5, width: 134.828125, height: 511}
```

The scrollbar appears when if I add:

```
  if (!props.slots.length) {
    return null;
  }
```

Seems like if component is mounted without slides the bug appears.

**Comment by WoodNeck** [comment]
Looks like it's a bug of Flicking that occurs when there're no panels but added after like you've said.
I'll fix this after, thanks for reporting!

**Comment by WoodNeck** [comment]
@matiosfree Sorry for the long wait!
I've just released @egjs/flicking 3.8.2(react-flicking 3.7.1) that fix this issue.
Try that and please notify me if that doesn't work :)

**Comment by matiosfree** [comment]
Yeah, it works for me now. Awesome work. Thanks :)
